src/main/java/cloud/fogbow/ras/core/plugins/interoperability/googlecloud/sdk/v1/network/models/InsertSubnetworkRequest.java

```diff
@@ -0,0 +1,61 @@
+package cloud.fogbow.ras.core.plugins.interoperability.googlecloud.sdk.v1.network.models;
+
+import cloud.fogbow.common.util.GsonHolder;
+import com.google.gson.annotations.SerializedName;
+
+import static cloud.fogbow.ras.core.plugins.interoperability.googlecloud.util.GoogleCloudConstants.Network.*;
```

Comment:
Better use import, instead of import static, since this will force you to explicitly refer to the static class name in the code, making it clearer that you are invoking a method from a static class. This leaves the code more readable and maintainable.